### PR TITLE
ensure base arguments are passed to all tests

### PR DESCRIPTION
**Pull request by ngoldbaum**
Thanks to @Xarthisius for the catch.

### CODE REVIEW

File: tests/nose_runner.py [hunk 1]
```diff
@@ -80,18 +80,21 @@ def generate_tasks_input():
...
+    base_argv = ['-s', '--nologcapture', '--with-xunit']
+
+    base_answer_argv = ['--local-dir=%s' % answers_dir, '--with-answer-testing',
+                        '--answer-big-data', '--local']
+
     args = []
 
     for test in list(tests["other_tests"].keys()):
         args.append(([test] + tests["other_tests"][test], True))
+        args.append(base_argv)
```
Comment (Xarthisius) [inline tests/nose_runner.py]: It think it should be:
```
args.append(([test] + base_argv + tests["other_tests"][test], True))
```

**Comment by Xarthisius** [comment]
@yt-fido test this please

**Comment by ngoldbaum** [comment]
@Xarthisius this correct now? It's hard to tell just from looking at the jenkins logs.

**Comment by Xarthisius** [comment]
It is. Now we have `unittests.xml` and `cookbook.xml` present in the workspace.

https://tests.yt-project.org/job/yt_py2_git/598/fingerprints/

**Pull request merged**